build.gradle

```diff
@@ -504,6 +504,10 @@ dependencies {
   implementation group: 'com.github.ben-manes.caffeine', name: 'caffeine', version: '3.1.8'
 
   implementation 'org.camunda.bpm:camunda-external-task-client:7.21.0'
+//  implementation group: 'javax.annotation', name: 'javax.annotation-api', version: '1.3.2'
```

Comment:
Shall we remove these commented lines?

Reply:
Removed - yep these are redundant now these were used for part of the attempt at local implementation/ with cftlib